src/help/technicalHistory.md

```diff
@@ -0,0 +1,48 @@
+#### January 18, 2017
```

Comment:
You should probably remove the dummy content to avoid confusion in that case, since it looks like actual content we need to keep for later. Maybe leave a note in there saying `First technical history release` or something like that.